src/app/src/test/kotlin/org/vaccineimpact/api/tests/models/CreateBurdenEstimateSetTests.kt

```diff
@@ -0,0 +1,100 @@
+package org.vaccineimpact.api.tests.models
+
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import org.vaccineimpact.api.models.BurdenEstimateSetType
+import org.vaccineimpact.api.models.BurdenEstimateSetTypeCode
+import org.vaccineimpact.api.models.CreateBurdenEstimateSet
+import org.vaccineimpact.api.serialization.validation.ValidationException
+
+class CreateBurdenEstimateSetTests : ValidationTests()
+{
+    @Test
+    fun `empty json throws validation exception`()
+    {
+        val badModelJson = "{}"
+        Assertions.assertThatThrownBy {
+            binder.deserialize<CreateBurdenEstimateSet>(badModelJson, CreateBurdenEstimateSet::class.java)
+        }
+
+        try
+        {
+            binder.deserialize<CreateBurdenEstimateSet>(badModelJson, CreateBurdenEstimateSet::class.java)
+        }
+        catch (e: ValidationException)
+        {
+            val errors = e.errors
+            Assertions.assertThat(errors.count()).isEqualTo(1)
+            Assertions.assertThat(errors[0].code).isEqualTo("invalid-field:create_burden_estimate_set:type:missing")
+        }
+    }
```

Comment:
I think the test structure you are looking for here is:

```
val badModelJson = "{}"
Assertions.assertThatThrownBy {
    binder.deserialize<CreateBurdenEstimateSet>(badModelJson, CreateBurdenEstimateSet::class.java)
}
        .isInstanceOf(ValidationException::class.java)
        .matches {
            val errors = (it as ValidationException).errors
            assertThat(errors).hasSize(1)
            assertThat(errors[0].code).isEqualTo("invalid-field:create_burden_estimate_set:type:missing")
            true
        }
```

Also, it might be good to factor this out, so you can write the tests like so:

```
assertThrowsThisMontaguError("invalid-field:create_burden_estimate_set:type:missing") {
      binder.deserialize(badModelJson, CreateBurdenEstimateSet::class.java)
}
```